app/views/articles/show.html.erb

```diff
@@ -31,6 +31,53 @@
     </script>
   <% end %>
 
+<script type="application/ld+json">
+  {
+    "@context": "http://schema.org",
+    "@type": "Article",
+    "mainEntityOfPage": {
+      "@type": "WebPage",
+      "@id": "https://dev.to/<%= article_url(@article) %>"
+    },
+    "url": "<%= article_url(@article) %>",
+    "image": "<%= article_social_image_url(@article) %>",
+    "publisher": {
+      "@context": "http://schema.org",
+      "@type": "Organization",
+      "name": "<%= community_qualified_name %>"
+      "logo": {
+        "@context": "http://schema.org",
+        "@type": "ImageObject",
+        "url": "<%= cloudinary(SiteConfig.logo_png, 192, "png") %>",
+        "width": "192",
+        "height": "192"
+      }
+    },
+    "headline": "<%= @article.title %>",
+    "author": {
+      "@context": "http://schema.org",
+      "@type": "Person",
+      "url": "<%= user_url(@user) %>",
+      "name": "<%= @user.name %>",
+      <% if @organization %>
```

Comment:
unfortunately this is tricky, as if there's no organization this is invalid JSON, see the trailing comma :(